glyphs-reader/src/font.rs

```diff
@@ -378,6 +326,19 @@ fn custom_param<'a>(
 }
 
 impl Font {
+    fn get_codepoints(&mut self, radix: u32) {
```

Comment:
usually if a method is called get_something I'd expect it to return something, in this case maybe would be better to name it "update_codepoints" or "parse_codepoints"?

Reply:
Oh good catch, I think originally it did return them but then I made it not. Renamed.